.github/workflows/packer-docker.yaml

```diff
@@ -0,0 +1,117 @@
+name: packer
+
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        type: string
+        required: false
+        default: '["ubuntu-latest"]'
+      ENVIRONMENTS:
+        description: '{"environments":[{"name":"dev"}, {"name":"qa"}]}'
+        required: true
+        type: string
+      IMAGE:
+        description: "Image to build Packer with"
+        required: true
```

Comment:
Github workflow is funny, even if you define "default" and add "required" it will still force the user to define the image. So you have to set `required: false` :(